### PR TITLE
Reverse first k elements in a queue in Python

### LINKED ISSUE #6545: Reverse First K elements of Queue in Python
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

## Motivation

(Please outline the motivation for the proposal.)

## Pitch

(Please explain why this feature should be implemented and how it would be used.)

### PR DESCRIPTION
**Pull request by arpit456jain**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

reversing first k elements in a queue
Enter the number of elements in array
5
Enter value of k
2
Enter the array
1 2 3 4 5

Output: 2 1 3 4 5

Fixes : #6545

### CODE REVIEW

**Comment by arpit456jain** [comment]
@ankitaggarwal23 kindly review  my PR

**Comment by arpit456jain** [comment]
@ankitaggarwal23

File: Python/README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 * [Rotate the array by k elements](ds/rotate_the_array_by_k_elements.py)
 * [Stack using list](ds/stack_using_list.py)
 * [Maximum distance between same elements](ds/MaxDistanceBetweenSameElements.py)
+* [Reverse First K elements of Queue.py](ds/Reverse_First_K_elements_of_Queue.py)
```
Comment (ankitaggarwal23) [inline Python/README.md]: remove .`py` from [Reverse First K elements of Queue.py]

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by arpit456jain** [comment]
@ankitaggarwal23 requested changes done

**Comment by arpit456jain** [comment]
@ankitaggarwal23

**Comment by arpit456jain** [comment]
@ankitaggarwal23

**Comment by arpit456jain** [comment]
@ankitaggarwal23 kindly merge this its been 14 days

**Comment by arpit456jain** [comment]
@ankitaggarwal23 @bhav09 can anyone review my PR its been a long time

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**